### PR TITLE
Change style of reslice cursor widget back to as it was in v25

### PR DESCRIPTION
**Pull request by bruyeret**
Remove lighting
Spin cyclinder representations to avoid cropping of the cylinder by the imageReslice

### CODE REVIEW

File: Sources/Widgets/Representations/LineHandleRepresentation/index.js [hunk 1]
```diff
@@ -96,6 +96,7 @@ function defaultValues(initialValues) {
     _pipeline: {
       glyph: vtkCylinderSource.newInstance({
         resolution: initialValues.glyphResolution ?? 4,
+        initAngle: initialValues.glyphAngle ?? 0,
```
Comment (finetjul) [inline Sources/Widgets/Representations/LineHandleRepresentation/index.js:99]: shouldn't it be PI/4 by default ?

Comment (bruyeret) [reply Sources/Widgets/Representations/LineHandleRepresentation/index.js:99]: In my opinion it is the responsibility of the reslice cursor to set the right angle for its use
The LineHandleRepresentation doesn't have a good reason to have a rotation by default

Comment (finetjul) [reply Sources/Widgets/Representations/LineHandleRepresentation/index.js:99]: The good reason is that the default resolution is 4.

If it was not 4, I agree it does not make sense to have an angle != 0.

But in this case it is explicit that we want a rectangular prism that is aligned with orthogonal axes.

No ?

Comment (bruyeret) [reply Sources/Widgets/Representations/LineHandleRepresentation/index.js:99]: The alignment with the axes is a requirement of the reslice cursor
Is it also a default requirement for the other widgets using this representation?

Comment (finetjul) [reply Sources/Widgets/Representations/LineHandleRepresentation/index.js:99]: I think it is a good default for ALL the lines, not just RCW.

Comment (bruyeret) [reply Sources/Widgets/Representations/LineHandleRepresentation/index.js]: Ok no problem, it is fixed!

**Pull request merged**